cmd/cdi/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/fsnotify/fsnotify v1.5.1
-	github.com/opencontainers/runtime-spec v1.1.0
+	github.com/opencontainers/runtime-spec v1.2.2-0.20250804081626-bfdffd548aa6
```

Comment:
If the new field is not yet in a new released spec version, then we could first remove the deprecated fields and then add the new field once a tag has been released.